cloudsmith/assets/dashboards/cloudsmith_overview.json

```diff
@@ -0,0 +1,455 @@
+                      "yaxis":{
+                         "include_zero":true,
+                         "scale":"linear",
+                         "label":"",
+                         "min":"auto",
+                         "max":"auto"
+                      },
+                      "markers":[
+                         
+                      ]
+                   },
+                   "layout":{
+                      "x":2,
+                      "y":2,
+                      "width":4,
+                      "height":2
+                   }
+                },
+                {
+                   "id":5049902402844905,
+                   "definition":{
+                      "title":"Total Bandwidth Consumed by Tokens",
+                      "title_size":"16",
+                      "title_align":"left",
+                      "show_legend":false,
+                      "legend_layout":"auto",
+                      "legend_columns":[
+                         "avg",
+                         "min",
+                         "max",
+                         "value",
+                         "sum"
+                      ],
+                      "time":{
+                         
+                      },
+                      "type":"timeseries",
+                      "requests":[
+                         {
+                            "queries":[
+                               {
+                                  "query":"avg:cloudsmith.token_bandwidth_total{*}",
+                                  "data_source":"metrics",
+                                  "name":"query1"
+                               }
+                            ],
+                            "response_format":"timeseries",
+                            "display_type":"line"
+                         }
+                      ],
+                      "yaxis":{
+                         "include_zero":true,
+                         "scale":"linear",
+                         "label":"",
+                         "min":"auto",
+                         "max":"auto"
+                      },
+                      "markers":[
+                         
+                      ]
+                   },
+                   "layout":{
+                      "x":6,
+                      "y":2,
+                      "width":4,
+                      "height":2
+                   }
+                }
+             ]
+          },
+          "layout":{
+             "x":0,
+             "y":9,
+             "width":12,
+             "height":5
+          }
+       },
+       {
+          "id":3664147694676081,
+          "definition":{
+             "type":"free_text",
+             "text":"Text Widget",
+             "color":"#4d4d4d",
+             "font_size":"auto",
+             "text_align":"left"
+          },
+          "layout":{
+             "x":0,
+             "y":0,
+             "width":2,
+             "height":1
+          }
```

Comment:
```suggestion
```
Seems like an unused widget

Reply:
Thanks @coignetp, I missed that sneaky widget. 

I have updated the PR.